src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -108,6 +108,18 @@
           type: string
           short-summary: Load balancer outbound IP prefix resource IDs.
           long-summary: Comma-separated public IP prefix resource IDs for load balancer outbound connection. Valid for Standard SKU load balancer cluster only.
+        - name: --load-balancer-outbound-ports
```

Comment:
This parameter receives one integer. Plural form may cause ambiguity.

Reply:
This property name comes from slb and we want to stay consistent with how they have named it there